Resources/Prototypes/Entities/Objects/base_item.yml

```
@@ -44,7 +44,7 @@
   - type: Pullable
   - type: DamageExaminable
   - type: TTS
-    voice: FactCore # Sunrise-TTS
+    voice: Portal2FactCore # Sunrise-TTS
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)